test/tests/bcs/dmg_periodic/tests

```diff
@@ -0,0 +1,58 @@
+[Tests]
+  issues = '#15501'
+  design = 'AddPeriodicBCAction.md  DistributedRectilinearMeshGenerator.md'
+
+  [./check]
+    type = 'RunException'
+    input = 'dmg_periodic_bc.i'
+    cli_args = '/UserObjects/uo/type=CheckGhostedBoundaries /UserObjects/uo/total_num_bdry_sides=160'
+    expect_err = 'All boundaries are ghosted to every single processor, and it is not scalable'
+    requirement = "MOOSE shall check ghosted boundaries"
```

Comment:
Should this be something more like:
"The system shall be able to determine if a non-scalable algorithm is being used for ghosting boundaries."
?

Reply:
You could ask @permcody to be sure, but I think we prefer "The system" over "MOOSE".

Reply:
Sometimes we use moose, and sometimes we use system .....

Reply:
The system seems to be more general.